openedx/core/djangoapps/api_admin/models.py

```diff
@@ -49,6 +48,19 @@ class ApiAccessRequest(TimeStampedModel):
 
     history = HistoricalRecords()
 
+    def __init__(self, *args, **kwargs):
+        super(ApiAccessRequest, self).__init__(*args, **kwargs)
+        # We need to dynamically set the help_text on the status field
+        # in order to use reverse -- otherwise, it's called before
+        # models are fully loaded and this make Django throw an
+        # exception.
+
+        # Translators: catalog_admin_url is HTML for a link to configure a user's catalogs.
+        self._meta.get_field('status').help_text = _(
```

Comment:
Use the [`fieldsets`](https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#django.contrib.admin.ModelAdmin.fieldsets) functionality instead of hacking the model.